src/ansys/pytwin/evaluate/twin_model.py

```diff
@@ -728,44 +744,44 @@ def get_geometry_filepath(self, rom_name: str):
         self._log_key = "GetGeometryFilePath"
 
         if not self.evaluation_is_initialized:
-            msg = "TwinModel has not been initialized! "
-            msg += "Please initialize evaluation before to call the geometry file getter!"
+            msg = "Twin model has not been initialized. "
+            msg += "Initialize evaluation before calling the geometry file getter."
             self._raise_error(msg)
 
         if self.tbrom_info is None:
-            self._raise_error("Twin model does not include any TBROM!")
+            self._raise_error("Twin model does not include any TBROM.")
 
         if rom_name not in self.tbrom_names:
-            msg = f"The provided rom_name {rom_name} has not been found in the available TBROM names. "
-            msg += f"Please call the geometry file getter with a valid TBROM name."
-            msg += f"\n Available TBROM name are: {self.tbrom_names}"
+            msg = f"The provided ROM name {rom_name} has not been found in the available TBROM names. "
+            msg += f"Call the geometry file getter with a valid TBROM name."
+            msg += f"\n Available TBROM names are: {self.tbrom_names}."
             self._raise_error(msg)
 
         filepath = os.path.join(self._tbrom_resource_directory(rom_name), "binaryOutputField", "points.bin")
 
         if not os.path.exists(filepath):
-            msg = f"Could not find the geometry file for given available rom_name: {rom_name}. "
-            msg += f"Geometry filepath you are looking for is: {filepath}"
+            msg = f"Could not find the geometry file for the given ROM name: {rom_name}. "
+            msg += f"The geometry filepath that you are looking for is: {filepath}."
             self._raise_error(msg)
 
         return filepath
 
     def get_rom_directory(self, rom_name):
         """
-        Get working directory path for a given Reduced Order Model (ROM) available in the TwinModel.
+        Get the working directory path for a ROM (reduced order model in the twin model.
 
         Parameters
         ----------
         rom_name : str
-            This is the name of a ROM model that is available in the TwinModel. See TwinModel.tbrom_names property to
-            get a list of available ROM model.
+            Name of the ROM. To get a list of available ROMs, see the
+            :attr:`pytwin.TwinModel.tbrom_names` attribute.
 
         Raises
         ------
         TwinModelError:
-            It raises an error if TwinModel has not been initialized.
-            It raises an error if TwinModel does not include any TBROM.
-            It raises an error if rom_name is not available.
+            If ``TwinModel`` has not been initialized.
```

Comment:
Same edits here in this line and the next.

Reply:
done. thnks